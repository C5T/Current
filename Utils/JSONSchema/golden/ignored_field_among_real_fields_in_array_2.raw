{"ObjectOrArray":{"fields":{"":{"ObjectOrArray":{"fields":{"x":{"ObjectOrArray":{"fields":{"a":{"String":{"values":{"foo":3},"instances":3,"nulls":0}},"b":{"Null":{"occurrences":3}},"c":{"String":{"values":{"bar":3},"instances":3,"nulls":0}}},"instances":3,"nulls":1}}},"instances":4,"nulls":0}}},"instances":1,"nulls":0}}