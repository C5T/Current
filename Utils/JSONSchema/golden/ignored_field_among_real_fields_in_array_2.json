[
  { "x": {
      "a": "foo",
      "b": null,
      "c": "bar"
    }
  },
  { "x": {
      "a": "foo",
      "b": [],
      "c": "bar"
    }
  },
  { "x": null },
  { "x": {
      "a": "foo",
      "c": "bar"
    }
  }
]
